quiz.py

```diff
@@ -10,11 +11,27 @@
 
 
 for i in range(10):
-    print(data['results'][i]['question'])
-    results = tuple(data['results'][i]['correct_answer'])
-    results += tuple([data['results'][i]['incorrect_answers']])
-    results = list(results)
-    random.shuffle(results)
-    print(results)
+    question = unescape(data['results'][i]['question'])  # Decode HTML entities
+    print(question)
+
+    # Combine correct and incorrect answers into a single list
+    correct_answer = [unescape(data['results'][i]['correct_answer'])]
+    answers = [unescape(data['results'][i]['correct_answer'])]
+    answers += [unescape(answer) for answer in data['results'][i]['incorrect_answers']]
+    random.shuffle(answers)
+    
+    
+    
+    print("here are the possible answers", answers)
+    response = input("what is your choice ? ")
+    
+    if (correct_answer[0] == response):
```

Comment:
it's great thank you, that even more than what I was hopping for. It's a nice idea to wait for the input and compare but sometimes it's a bit to long to answer by copying the entire sentence.